packages/styles/src/theming/sap_fiori_3.scss

```diff
@@ -161,6 +161,7 @@
   --fdList_Button_Spacing: 0.125rem;
   --fdList_Growing_Font_Weight: normal;
   --fdList_Active_Outline_Color: var(--sapContent_ContrastFocusColor);
+  --fdList_Unread_Indicator_Spacing: 2.125rem;
```

Comment:
why is this a theming parameter when its value is the same in all themes?

Reply:
It is used across multiple files. Like for byline list, navigation list item, no border list item

Reply:
that's not a good reason. If it's used in multiple files, it should be placed somewhere where all files have access to it. Imagine the value needs to be updated. You have to go to all these files and update it.

Reply:
in the theming files we should have only variables that concern the theming delta